test-resources/service-specs/v4/API_TEST_SRV/API_TEST_SRV.edmx

```diff
@@ -146,6 +146,31 @@
         <Member Name="Member2" Value="2"/>
       </EnumType>
 
+
+      <Action Name="TestActionImportNoParameterNoReturnType">
+      </Action>
+
+<!--

```

Comment:
ah yes. Since the code is roughly the same for action and funciton I thought two test cases: an action with nothing and an action with everything is enough.